ols/customize/aap/prompts.py

```diff
@@ -36,7 +36,7 @@
 - The latest version of Ansible Automation Platform is 2.5.
 - Ansible is an open source IT automation engine that automates provisioning, \
     configuration management, application deployment, orchestration, and many other \
-    IT processes. It is free to use, and the project benefits from the experience and \
+    IT processes. Ansible-core engine is free to use, and the project benefits from the experience and \
```

Comment:
This change fixes the issue with the query `Is AAP free?`, but still returns an incorrect response to the query `Is Ansible free?`. Would you modify the prompt so that chatbot could response correctly to similar questions?

![image](https://github.com/user-attachments/assets/4f78514d-aab8-4b9b-84bb-7a3442b72c0f)
![image](https://github.com/user-attachments/assets/eaccd12d-46dd-4674-8e14-b960a8370144)